src/operator/config/rbac/manifests-patched.yaml

```diff
@@ -253,6 +253,19 @@ rules:
   - patch
   - update
   - watch
+- apiGroups:
```

Comment:
Why is this here? Seems unrelated to this PR?
@evyatarmeged possibly affected by the Linkerd PR, could indicate something is missing there?